### PR TITLE
impr(doc): New doc website and readme about new events

### PR DESCRIPTION
**Pull request by hereismass**
### Description

Doc website has been revamped entirely, with a better demo.

Readme updated with new events

Before:
capture d ecran 2018-10-11 a 18 18 24

Now:
capture d ecran 2018-10-11 a 18 18 10

### Submission

- [x] Your branch is based on `master`
- [x] Your code is unit tested
- [x] CircleCI builds are green (coverage and lint)
- [x] Add team "CTI" as "Reviewer" for the PR
- [x] Name your PR with the convention `fix|feat|impr(<subject>): description`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,8 +63,58 @@ You can send messages to the phone and listen messages coming from it.
...
+    communication_id: 'abcdeABCDE12345'
+  }
+  ```
+- `external_dial`: a dial has been made from outside of the phone (api/extension)
+  ```javascript
+  {
+    phone_number: '+15557543010';
+  }
+  ```
+- `powerdialer_updated`: a powerdialer campain has been updated (via extension). There is no payload.
```
Comment (eulbat) [inline README.md]: campain -> campaign

File: demo/index.html [hunk 2]
```diff
@@ -1,189 +1,183 @@
...
+      <a id="phone-aircall" class="d-none">
+        <img src="images/aircall-icon.svg">
+      </a>
+    </div>
+  </nav>
+
+  <main role="main">
+    <div class="jumbotron">
+      <h1>Aircall Everywhere</h1>
+      <p>Aircall Everywhere purpose is to let you embed Aircall phone into your website and interacts with it. If
```
Comment (eulbat) [inline demo/index.html]: interacts -> interact

File: demo/demo.js [hunk 3]
```diff
@@ -1,79 +1,178 @@
...
+import './reset.scss';
+import './demo.scss';
+
 console.log('demo time!');
 
+// Show phone
+const setPhoneVisibility = visible => {
+  const phoneContainer = document.querySelector('#phone-container');
+  if (!!visible) {
+    phoneContainer.classList.remove('d-none');
```
Comment (Shinomix) [inline demo/demo.js]: `phoneContainer.classList.toggle('d-none')`?

Comment (hereismass) [reply demo/demo.js]: didnt know it existed... Thanks!

But here we dont always want to toggle, like when receiving a call we want to show the phone, so we call it even if it is already open

File: README.md [hunk 4]
```diff
@@ -63,8 +63,58 @@ You can send messages to the phone and listen messages coming from it.
...
+  {
+    from: '+15557543010',
+    to: '+15551234567',
+    communication_id: 'abcdeABCDE12345'
+  }
+  ```
+- `outgoing_answered`: an outgoing call has been answered
+  ```javascript
+  {
+    communication_id: 'abcdeABCDE12345';
```
[the remark this answers is not in the document]
Comment (hereismass) [reply README.md]: prettier fucked up on this :D

**Pull request merged**